### DISCUSSION TITLE
Battery health dashboard: Add "Number of 100% charges" and "Number of <20% charges"

### DISCUSSION THREAD

**Discussion by KrzysztofHajdamowicz** [teslamate-org/teslamate#4145] [Ideas]
I'd like to see battery health dashboard expanded with two additional fields:

1. Number of times, where battery reached 100%
2. Number of times, where battery stayed below 20%

That would require to perform SELECT on whole known car history in postgres database.

**Comment by stauffenberg2020** [comment]
Is it already not available in the charging stats dashboard?

**Comment by jheredianet** [comment]
This discussion should be closed. Charging stats dashboard is exactly for that.
